### ISSUE TITLE
fails to generate bower_components

### ISSUE THREAD

**Issue by zabaat** [konstantinzolotarev/sails-hook-adminpanel#47]
Sails 0.12.1
Node 5.9.0
Ubuntu 14

when going to /admin I get the following

> http://zabaat.com:1337/admin/bower_components/iron-icons/iron-icons.html
> Failed to load resource: the server responded with a status of 404 (Not Found)

> menu-collapse.html:21 Uncaught ReferenceError: Polymer is not defined

![image](https://cloud.githubusercontent.com/assets/3325067/13902195/903cb646-edfc-11e5-9078-c867febb8a7a.png)

![image](https://cloud.githubusercontent.com/assets/3325067/13902198/9ff0bfba-edfc-11e5-9f58-284277367e57.png)

> sails> verbose: Grunt :: Running "bower:install" (bower) task
> verbose: Grunt :: >> Cleaned target dir /var/www/poe/server/assets/vendor
> verbose: Grunt :: >> Installed bower packages
> verbose: Grunt :: >> Copied packages to /var/www/poe/server/assets/vendor
> verbose: Grunt :: >> Cleaned bower dir /var/www/poe/server/bower_components
> verbose: Grunt :: Running "clean:dev" (clean) task

Doesn't seem like it's putting the files in the right place? the only files in assets/vendor are the bootstrap and jquery files.

**Comment by dilame** [comment]
Yes, the same issue, bratan.

**Comment by konstantinzolotarev** [comment]
@dilame Same OS ? 
Tested with OSX - working. Will check with Ubuntu 14 today evening.

**Comment by dilame** [comment]
@konstantinzolotarev Ubuntu 14.04

**Comment by zabaat** [comment]
From the chrome debug:

http://localhost:1338/admin/bower_components/iron-icons/iron-icons.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/iron-collapse/iron-collapse.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/paper-header-panel/paper-header-panel.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/iron-icon/iron-icon.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/paper-toolbar/paper-toolbar.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/paper-menu/paper-menu.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/paper-toast/paper-toast.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/paper-material/paper-material.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/paper-item/paper-item.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/paper-icon-button/paper-icon-button.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/paper-fab/paper-fab.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/paper-button/paper-button.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/paper-input/paper-input.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/paper-input/paper-textarea.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/paper-checkbox/paper-checkbox.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/gold-email-input/gold-email-input.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/material-data-table/material-data-table.html Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:1338/admin/bower_components/paper-styles/paper-styles.html Failed to load resource: the server responded with a status of 404 (Not Found)
menu-collapse.html:21 Uncaught ReferenceError: Polymer is not defined
admin-menu-item.html:45 Uncaught ReferenceError: Polymer is not defined
admin-alert.html:41 Uncaught ReferenceError: Polymer is not defined
app-checkbox.html:18 Uncaught ReferenceError: Polymer is not defined

**Comment by zabaat** [comment]
Workaround:
still not sure why it doesn't auto copy, but at least I have a 'working' admin panel

```
cd ./node_modules/sails-hook-adminpanel
bower install
choose option 7 when prompted (I have no idea if this is the best)
//this creates the necessary files in     node_modules/sails-hook-adminpanel/assets
 copy this directory to your ./assets/admin  folder in the root of your sails directory.
```

**Comment by dilame** [comment]
@zabaat it's better to use previuos version of admin panel, till @konstantinzolotarev fix this problem.

**Comment by konstantinzolotarev** [comment]
@dilame @zabaat Published new version `1.0.15` with fix for this problem.
Please verify.

**Comment by zabaat** [comment]
@konstantinzolotarev  yes thank you! that worked! 
Sails 0.12.3 on ubuntu 14.04
